ssdc-rm-common-entity-model/src/main/java/uk/gov/ons/ssdc/common/model/entity/UacQidLink.java

```diff
@@ -19,6 +19,9 @@
 @TypeDefs({@TypeDef(name = "jsonb", typeClass = JsonBinaryType.class)})
 @Entity
 @DynamicUpdate
+@Table(
+    name = "uac_qid_link",
+    indexes = {@Index(name = "uac_qid_idx", columnList = "qid")})
```

Comment:
Small point but a more precise name on the index might be helpful, `uac_qid_idx` isn't clear which column it refers to